tests/dummy/app/templates/application.hbs

```diff
@@ -133,11 +133,32 @@
     {{github-issue title="Focus Ordering"}}
 
     <h3>Query Params</h3>
-    <p>Scenarios to write:</p>
-    <ul>
-      <li>Refreshing model</li>
-      <li>Non-refreshing model</li>
-    </ul>
+    <p>These tests are for focusing behavior. You may use your mouse to click through the links in this test.</p>
+    <ol>
+      <li><button {{action "resetDemo"}}>Reset Demo</button></li>
+      <li>{{#link-to "feed"}}Load the Feed route.{{/link-to}}</li>
+      <li>Focus should be set to a block which begins with the "Feed" header.</li>
+
+      <li>{{#link-to "feed" (query-params sort="title")}}Update query params to sort by title.{{/link-to}}</li>
+      <li>Model was <em>not</em> refreshed; full transition <em>not</em> performed.</li>
+      <li>Items should reorder into alphabetical order.</li>
+      <li>No focus should be set.</li>
```

Comment:
@nathanhammond Are you referring to the way that the focus _isn't_ being set if the model doesn't reload / route doesn't perform a full transition -- or the way that the focus _is_ being set if they do?

My first intuition was that the focus should be set for both cases, but I've come around to the notion that the current behavior affords an important distinction for the way data is driving the UI: In the first case (no model reload), we're saying that we're cool with the UI updating on its own and -- literally and figuratively -- we're not _particularly_ focused on what happened. In the second case, though, a model reload & transition is almost a way of saying that we _are_ focused on what's happening, and so having the outlet focus makes sense.